thoth/storages/graph/postgres.py

```diff
@@ -4616,7 +4624,7 @@ def _get_or_create_python_package_index(
         if python_package_index is None:
             if only_if_enabled:
                 raise PythonIndexNotRegistered(f"Python package index {index_url!r} is not know to system")
-
+            index_url = GraphDatabase.map_python_index_url(index_url)
```

Comment:
Why not at the beginning of the method? :)

Reply:
Ah right! Changed it :)